.github/CODEOWNERS

```diff
@@ -141,15 +143,22 @@
 
 
 # Global components and filters.
-/core/templates/dev/head/components/alerts/ @ankita240796
-/core/templates/dev/head/components/attribution_guide/ @ankita240796
-/core/templates/dev/head/components/background/ @ankita240796
-/core/templates/dev/head/components/embed_modal/ @ankita240796
-/core/templates/dev/head/components/loading/ @ankita240796
+/core/templates/dev/head/components/common-layout-directives/common-elements/alert-message.directive.ts @ankita240796
```

Comment:
You don't need to do this file by file -- it's hard to maintain. Instead, it's fine to assign someone to a directory, and in fact, hopefully what you're doing with the frontend refactoring helps to streamline this. So, please try to use directories where possible.

Reply:
But there are many files in a directory that do not have the same CODEOWNER. For example in button-directives/ the create-activity button, embed, and hint-and-solution belong to different CODEOWNERS.

Reply:
Yep, in that case, you can use individual filepaths. I'm only talking about cases where every file in a directory has the same codeowner.

Reply:
Yes I have included the directories where all the CODEOWNERS match (e.g. common-layout-directives/navigation-bars). Only those with mismatching CODEOWNERS are named by filepaths